test/mocks/MockHttp.scala

```diff
@@ -33,6 +33,7 @@ import scala.concurrent.{ExecutionContext, Future}
 
 trait MockHttp extends UnitSpec with BeforeAndAfterEach {
 
+  //todo: refactor: rename mockHttpGet to mockHttp
```

Comment:
Was there a reason for not doing this?

Reply:
I was planning to do any refactoring needed as a separate PR at the end as it's not specifically related to this ticket, including may be the isCrystalised Boolean ticket

Reply:
Cool